src/components/workspace/BuildProject/BuildProject.tsx

```diff
@@ -638,9 +644,29 @@ const BuildProject: FC<Props> = ({ projectId, contract, updateContract }) => {
     if (activeProject?.network) {
       setEnvironment(activeProject.network);
     }
-    if (activeProject?.selectedContract) {
-      setSelectedContract(activeProject.selectedContract);
-      deployForm.setFieldsValue({ contract: activeProject.selectedContract });
+    const selectedABIPath = activeProject?.selectedContract;
```

Comment:
Abbreviations in camel case are handled as regular words.

[Here's a review](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase) of the reasoning behind it.

The most important part there is that automated tools can only work if rules do not depend on semantics. For example, if there would be a file that exports `selectedABIPath`, its name by Node.js file naming standards would be `selected-a-b-i-path`, and linter would enforce that.